requirements.txt

```diff
@@ -1,6 +1,6 @@
 sentry-sdk[flask]
 uwsgi
 uwsgi-tools
-# MarkupSafe needs to be pinned to version used by superset itself otherwise functionality breaks
```

Comment:
@HusnaHariz does it work okay without this?

Reply:
So far no issues observed. Will keep observing.